tests/testthat/test-test_footer_dfe_auto.R

```diff
@@ -0,0 +1,66 @@
+# create output for testing
+
+output <- dfeshiny::footer_dfe(full = FALSE)
+
+
+test_that("outputs are as expected", {
```

Comment:
Thanks for adding these. Next up, it'd be good to also add an example of it being implemented in the test dashboard:

https://github.com/dfe-analytical-services/dfeshiny/blob/main/tests/test_dashboard/ui.R